docs/en/observability/monitor-servers.asciidoc

```diff
@@ -172,3 +172,44 @@ To drill down and analyze the metric anomaly, select *Actions > Open in Anomaly
 Inventory page, filtered by the specific metric. 
 ====
 
+[%collapsible]
+.*Osquery*
+====
+
+[IMPORTANT]
+=====
+You must have an active {fleet-guide}/elastic-agent-installation.html[{agent}] with an assigned agent policy
+that includes the https://docs.elastic.co/en/integrations/osquery_manager.html[Osquery Manager]
```

Comment:
This would also be a good link to update once https://github.com/elastic/kibana/pull/109885/ is merged, so that it goes to something more like `https://www.elastic.co/guide/en/kibana/current/osquery.html`